### PR TITLE
feat: refine the setup page.

### PR DESCRIPTION
**Pull request by stu01509**
Refine the setup page.

Fixes #

#### Short description of what this resolves:
Remove outdated code and inline-style.
Adjust the Bootstrap grid and button style.

image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: setup.php [hunk 1]
```diff
@@ -299,9 +327,9 @@ function recursive_writable_directory_test($dir)
         background-color:  WHITESMOKE;
         padding:0 10px;
     }
-    button {
+    /* button {
     font-weight:bold;
-    }
+    } */
```
Comment (bradymiller) [inline setup.php:332]: better to remove code than comment out

File: setup.php [hunk 2]
```diff
@@ -133,43 +133,71 @@ function recursive_writable_directory_test($dir)
...
-                <div class="row">
-                <div class="col-sm-12">
-                    <div class="mb-3 border-bottom">
-                        <h2>OpenEMR Setup <a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="Click to view Help"><i class="fa fa-question-circle" aria-hidden="true"></i></a></h2>
-                    </div>
-                </div>
-            </div>
+        <nav class="navbar navbar-expand navbar-light bg-light">
+            <a class="navbar-brand" href="#">OpenEMR Setup</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (tywrenn) [inline setup.php:138]: Replace the default id from the BS4 doc `navbarSupportedContent` with something else

File: setup.php (hunk 1 above)
Comment (tywrenn) [reply setup.php:332]: Yepppp

File: setup.php [hunk 3]
```diff
@@ -299,9 +327,9 @@ function recursive_writable_directory_test($dir)
         background-color:  WHITESMOKE;
```
Comment (tywrenn) [inline setup.php:327]: Remind me to clean this CSS up

File: setup.php [hunk 4]
```diff
@@ -324,21 +352,27 @@ function cloneClicked() {
...
-    <div class = 'mt-4 container'>
-        <div class="row">
-            <div class="col-sm-12">
-                <div class="mb-3 border-bottom">
-                    <h2>OpenEMR Setup <a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="Click to view Help"><i class="fa fa-question-circle" aria-hidden="true"></i></a></h2>
-                </div>
-            </div>
+    <nav class="navbar navbar-expand navbar-light bg-light">
+        <a class="navbar-brand" href="#">OpenEMR Setup</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (tywrenn) [inline setup.php:357]: Replace the default id from the BS4 doc `navbarSupportedContent` with something else

Comment (tywrenn) [reply setup.php:357]: Also it's bad to have 2 id's that are the same

File: setup.php [hunk 5]
```diff
@@ -433,77 +471,92 @@ function cloneClicked() {
...
-                                <label for='inst1'>
-                                <input checked id='inst1' name='inst' type='radio' value='1'>Have setup create the database
-                                </label><br />
-                                <label for='inst2'>
-                                <input id='inst2' name='inst' type='radio' value='2'>I have already created the database
-                                </label><br />
+                                <input name='state' type='hidden' value='2' />
+                                <input name='site' type='hidden' value='$site_id' />
+                                <div class="form-check">
+                                    <input checked class='form-check-input' id='inst1' name='inst' type='radio' value='1'>
```
Comment (tywrenn) [inline setup.php:484]: Slash to close the input

File: setup.php [hunk 6]
```diff
@@ -433,77 +471,92 @@ function cloneClicked() {
...
+                                    <label class="form-check-label" for="inst1">
+                                        Have setup create the database
+                                    </label>
+                                </div>
                                 <br />
-                                <button type='submit' value='Continue'><b>Proceed to Step 2</b></button>
-                            </form><br />
-                        </fieldset>
+                                <div class="form-check">
+                                    <input id='inst2' class='form-check-input' name='inst' type='radio' value='2'>
```
Comment (tywrenn) [inline setup.php:491]: Slash to close the input

**Comment by bradymiller** [comment]
The pre-install screen is a bit hard to follow because of the spacing.
1. The related stuff below Ensuring following file is world-writable.. should have less space between it so it is more obvious that it is related to it (maybe even a margin or something to it also to make it more clear)
2. Same thing the related stuff below Ensuring the directories have proper permissions...

File: setup.php [hunk 7]
```diff
@@ -1349,147 +1410,168 @@ function cloneClicked() {
...
+                        &nbsp;&nbsp;&lt;/Directory&gt;</code><br /><br />";
 
                         $btn_text = 'Proceed to Select a Theme';
                         $step5_bottom = <<<STP5BOT
-                        <p>If you are having difficulty finding your apache configuration file, then refer to the <a href='Documentation/INSTALL' rel='noopener' target='_blank'><span style='text-decoration: underline;'>'INSTALL'</span></a> manual for suggestions.</p>
+                        <p>If you are having difficulty finding your apache configuration file, then refer to the <a href='Documentation/INSTALL' rel='noopener' target='_blank'><u>'INSTALL'</u></a> manual for suggestions.</p>
                         <p>We recommend you print these instructions for future reference.</p>
                         <p class='mark'>Click <strong>'$btn_text'</strong> to select a theme.</p>
                         <br />
                         <form method='post'>
```
Comment (stu01509) [inline setup.php:1459]: Hi @tywrenn 

Should I replace the `.mark` class to HTML mark tag?

**Comment by stu01509** [comment]
Update the commit.

**Comment by bradymiller** [comment]
the pre-install still has spacing issues. Would shoot for spacing like this:
```
Ensuring following file is world-writable...
    /var/www/localhost/htdocs/openemr/sites/default/sqlconf.php file is ready.

Ensuring the /var/www/localhost/htdocs/openemr/sites/default/documents directory and its subdirectories have proper permissions...
    /var/www/localhost/htdocs/openemr/sites/default/documents directory and its subdirectories are ready.

All required files and directories have been verified.
```

**Comment by bradymiller** [comment]
This may be something that is better for @ywrenn to weigh in on:
1. it seems like pulling the buttons to the left rather than the right would be more consistent with OpenEMR current look and feel.
2. The top left/right OpenEMR setup and Help are outside of the view of my eyeballs when running setup:

image

**Comment by stu01509** [comment]
Hi @bradymiller 

Thanks for your review, I adjust that a little bit, please take a look :)

image


cc @tywrenn

**Comment by stephenwaite** [comment]
hi @stu01509 , what if the `text-success` for `Ensuring...` was `text-info`? that way could have 3 different colors there?

File: setup.php (hunk 7 above)
Comment (tywrenn) [reply setup.php:1463]: I would leave it alone due to differing styles between the p tag and the mark tag

**Comment by tywrenn** [comment]
@stephenwaite I don't really see us using `text-info` that much (in fact we don't really use it) so to prevent color clashing i'd just leave it there

**Comment by stephenwaite** [comment]
hi @tywrenn , i can bring it in unless there's something for @bradymiller to review?

**Comment by bradymiller** [comment]
hi @stu01509 , Looks great and testing well. Bringing in. Thanks for yet another awesome improvement! -brady
